src/software/ai/passing/cost_function_test.cpp

```diff
@@ -422,21 +402,6 @@ TEST_F(PassingEvaluationTest, ratePassShootScore_no_robots_and_directly_facing_g
     EXPECT_GE(1, pass_shoot_score);
 }
 
-TEST_F(PassingEvaluationTest,
-       DISABLED_ratePassShootScore_no_robots_and_facing_directly_away_from_goal)
-{
-    // No robots on the field, we receive the pass and are facing directly away from the
-    // goal
-    Team enemy_team(Duration::fromSeconds(10));
-    enemy_team.updateRobots({});
-    Field field = Field::createSSLDivisionBField();
-    Pass pass({-1, 0}, {0, 0}, 1);
-
-    double pass_shoot_score = ratePassShootScore(field, enemy_team, pass, passing_config);
-    EXPECT_GE(pass_shoot_score, 0.0);
-    EXPECT_LE(pass_shoot_score, 0.05);
-}
-
```

Comment:
why delete this test?

Reply:
It didn't really provide much value to test no robots being on the field and I wasn't sure what the expected rating should be, and the test was also already disabled.